internal/connector/common/resources_common.go

```diff
@@ -50,3 +73,152 @@ func GenerateCommentInformation(data map[string]string) string {
 
 	return commentInformation
 }
+
+func CheckSingletonResource(response *http.Response, err error, apiFuncName, resourceType string) (bool, error) {
+	ok, err := HandleClientResponse(response, err, "ReadBrandingSettings", resourceType)
+	if err != nil {
+		return false, err
+	}
+	if !ok {
+		return false, nil
+	}
+
+	if response.StatusCode == 204 {
+		output.Warn("API client 204 No Content response.", map[string]interface{}{
+			"api_function": apiFuncName,
+			"resource":     resourceType,
+		})
+		return false, nil
+	}
+
+	return true, nil
+}
+
+func GetManagementAPIObjectsFromIterator[T any](iter management.EntityArrayPagedIterator, clientFuncName, extractionFuncName, resourceType string) ([]T, error) {
```

Comment:
Seems like these would fit better inside the `pingone` folder

Reply:
restructured